src/test/java/duke/task/TaskTest.java

```diff
@@ -0,0 +1,63 @@
+package duke.task;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import org.junit.jupiter.api.Test;
+
+import duke.exceptions.DukeException;
+
+public class TaskTest {
+    @Test
+    public void testSetDone() throws DukeException {
+        Todo todo = new Todo("task");
+        todo.setDone();
+        assertEquals(true, todo.getDoneStatus());
+
+    }
+
+    @Test
+    public void testDeleteTask() throws DukeException {
```

Comment:
Perhaps you can separate the tests and name your test methods in this format: featureUnderTest_testScenario_expectedBehavior().